src/main/kotlin/com/chattriggers/ctjs/api/message/TextComponent.kt

```diff
@@ -104,7 +109,7 @@ class TextComponent : Text {
     fun setClickAction(value: Any?) = apply {
         clickAction = when (value) {
             is ClickEvent.Action -> value
-            is String -> ClickEvent.Action.byName(value)
+            is String -> ClickEvent.Action.byName(value.lowercase())
```

Comment:
Can you put these in a separate commit? In general if you write "Also, do xyz" anywhere in your commit message, that's a good sign that whatever you're talking about should be in its own commit.